### DISCUSSION TITLE
Unable to log to the console after (I think) a successfull installation

### DISCUSSION THREAD

**Discussion by Lsm-CVX** [zitadel/zitadel#8261] [Q&A]
HI all, 

I am struggling with some configuration. I have install ZITADEL on a LXC alpine 3.20 and is working jsut fine on the console but I can not connect to it from an other computer from the LAN.

Here is below the configuration : 

config.yaml
`
ExternalDomain: 10.0.0.18
ExternalPort: 8080
ExternalSecure: false

Database:
  EventPushConnRatio: 0.2 # ZITADEL_DATABASE_COCKROACH_EVENTPUSHCONNRATIO
  ProjectionSpoolerConnRatio: 0.2 # ZITADEL_DATABASE_COCKROACH_PROJECTIONSPOOLERCONNRATIO
  postgres:
    Host: localhost # ZITADEL_DATABASE_POSTGRES_HOST
    Port: 5432 # ZITADEL_DATABASE_POSTGRES_PORT
    Database: zitadel # ZITADEL_DATABASE_POSTGRES_DATABASE
    Options: # ZITADEL_DATABASE_POSTGRES_OPTIONS
    User:
      Username: zitadel # ZITADEL_DATABASE_POSTGRES_USER_USERNAME
      Password: zitadel # ZITADEL_DATABASE_POSTGRES_USER_PASSWORD
      SSL:
        Mode: disable # ZITADEL_DATABASE_POSTGRES_USER_SSL_MODE
    Admin:
      ExistingDatabase: postgres# ZITADEL_DATABASE_POSTGRES_ADMIN_EXISTINGDATABASE
      Username: postgres # ZITADEL_DATABASE_POSTGRES_ADMIN_USERNAME
      Password: # ZITADEL_DATABASE_POSTGRES_ADMIN_PASSWORD
      SSL:
        Mode: disable# ZITADEL_DATABASE_POSTGRES_ADMIN_SSL_MODE
`

```
zitadel:~# zitadel --config config.yaml --masterkey "MasterkeyNeedsToHave32Characters" --tlsMode disabled start
  _____  ___   _____      _      ____    _____   _
 |__  / |_ _| |_   _|    / \    |  _ \  | ____| | |
   / /   | |    | |     / _ \   | | | | |  _|   | |
  / /_   | |    | |    / ___ \  | |_| | | |___  | |___
 /____| |___|   |_|   /_/   \_\ |____/  |_____| |_____|

 ===============================================================

 Version          : v2.55.2
 TLS enabled      : false
 External Secure  : false
 Console URL      : http://10.0.0.185:8080/ui/console
 Health Check URL : http://10.0.0.185:8080/debug/healthz

 Warning: you're using plain http without TLS. Be aware this is
 not a secure setup and should only be used for test systems.
 Visit: https://zitadel.com/docs/self-hosting/manage/tls_modes

 ===============================================================
```

ERRO[0028] unable to set instance                        caller="/home/runner/work/zitadel/zitadel/internal/api/http/middleware/instance_interceptor.go:60" error="unable to get instance by host 10.0.0.18:8080: ID=QUERY-1kIjX Message=Errors.IAM.NotFound" externalDomain=10.0.0.185 origin="http://10.0.0.18:8080"

Whereas : 
zitadel:~# curl http://localhost:8080/debug/healthz
ok

and on computer on the LAN return : 

unable to set instance using origin http://10.0.0.18:8080 (ExternalDomain is 10.0.0.185): ID=QUERY-1kIjX Message=IAM non trouvé. Assurez-vous que vous avez la bonne organisation. Vérifiez https://zitadel.com/docs/apis/introduction#organizations

I am guessing that the ExternalDomain: 10.0.0.18 is the culprit but I did not figure out how to make it works. 

I tried also the NGINX install but it failed the same way. 

Is there an issue I did not catch ? 

Thanks 
LSM

**Comment by hifabienne** [comment]
@eliobischof @stebenz can you help here?

**Reply by Lsm-CVX** [reply]
Quick update. 
I did the same suing docker and go the same issue, I can not connect using a computer. 

Looking forward to hearing form you

Laurent

**Reply by williamaib** [reply]
Hi @Lsm-CVX Did you find the solution to this Issue

**Comment by HungryHowies** [comment]
Hey @Lsm-CVX 

 I noticed you have two different IP address?

 10.0.0.18:8080: ID=QUERY-1kIjX Message=Errors.IAM.NotFound" externalDomain=10.0.0.185

**Reply by Lsm-CVX** [reply]
Hi HungryHowies,

Yes of course , the serveur is 10.0.0.18, my station is 10.0.0.185. That is my problem because I will need to have access to the console remotely

**Reply by stebenz** [reply]
Is there a reason why you use an IP as the externalDomain?

**Reply by Lsm-CVX** [reply]
Yes, I want to have the ingress on one IP and the authentifaction middelware on an other. It is our way of working. But maybe I do not understand the core of the ExternalDomain. 

is there a way to do it like that ? 

anyway the sever is headless so i will never be hable to connect to it directly.

**Reply by Lsm-CVX** [reply]
@stebenz : We do not have an external domain available on the LAN (it is something like toto.infra, the DNS and the reserve is functionnal but the domain is not routed outside.

**Comment by williamaib** [comment]
Been Getting the same error, can someone please help not able to resolve this, tried so many ways.

**Reply by fforootd** [reply]
Can you please share how you setup and deployed Zitadel and also what error you encounter in detail.

**Reply by williamaib** [reply]
Hi, I'm discussing on the same in your Discord Server. Have provided the info there

**Reply by fforootd** [reply]
Got it, would you be so kind and link it here so that I can tie that together 😁

**Reply by williamaib** [reply]
https://discord.com/channels/927474939156643850/927866013545025566/1311606072137744395

**Reply by fforootd** [reply]
To reduce redundant communication I am going to lock this discussion.

But for everyone who comes across this issue. This has likely to do with the reverse-proxy configuration.

For guides check our [docs](https://zitadel.com/docs/self-hosting/manage/reverseproxy/reverse_proxy)

And the discord chat about this [here](https://discord.com/channels/927474939156643850/927866013545025566/1311634975547854878) and [here](https://discord.com/channels/927474939156643850/927866013545025566/1311669558897147925)
